indexes/about-self.js

```diff
@@ -82,6 +82,8 @@ module.exports = class AboutSelf extends Plugin {
       profile.image = content.image.link
     else if (typeof content.image === 'string') profile.image = content.image
 
+    if (content.publicWebHosting !== 'undefined')  profile.publicWebHosting = content.publicWebHosting
+    
```

Comment:
This would require existing index entries to be updated, so we have to bump up the index's "version". You do that by changing the number `3` to `4` on line 20.

Reply:
@rabble we still need the index version bumped.